unittests/Feature/FeatureModel.cpp

```diff
@@ -13,6 +13,79 @@ TEST(FeatureModel, build) {
   EXPECT_TRUE(B.buildFeatureModel());
 }
 
+TEST(FeatureModel, cloneUnique) {
+  auto FM = FeatureModelBuilder().buildSimpleFeatureModel(
+      {{"a", "aa"}, {"root", "aba"}, {"root", "a"}});
+  assert(FM);
+
+  auto Clone = FM->clone();
+
+  assert(Clone);
+  for (const auto &Feature : FM->features()) {
+    EXPECT_NE((void *)Clone->getFeature(Feature->getName()), (void *)Feature);
+  }
+}
+
+TEST(FeatureModel, cloneRoot) {
+  FeatureModelBuilder B;
+  BinaryFeature A("a");
+  B.addFeature(A);
+  B.setRoot("a");
+  auto FM = B.buildFeatureModel();
+  assert(FM);
+
+  auto Clone = FM->clone();
+  assert(Clone);
+  FM.reset();
+
+  // NOLINTNEXTLINE
+  ASSERT_EXIT(EXPECT_TRUE(FM->getFeature("a")->isRoot()),
+              testing::KilledBySignal(SIGSEGV), ".*");
```

Comment:
Why do you need the EXPECT_TRUE inside the EXIT? Should the segfault not also occur without it?